src/Codeception/Module/Asserts.php

```diff
@@ -15,10 +15,11 @@ class Asserts extends CodeceptionModule
      * @param        $expected
      * @param        $actual
      * @param string $message
+     * @param float  $delta
```

Comment:
Please add some documentation about what delta is and how to use it. Add it to this docblock.